### PR TITLE
Split starting game from started game

### PR DESCRIPTION
**Pull request by agustinesco**
Now there will be a new state between the creation/joining of the lobby and the start of the game called "preparing_game" which will remain until the runner starts the tick of the game

THIS PR SHOULD BE TESTED AND MERGED ALONSIDE PR NUMBER 1214 FROM THE CURSE OF MYRRA REPO Y ORDER TO WORK

### CODE REVIEW

File: load_test/lib/load_test/communication/messages.pb.ex [hunk 1]
```diff
@@ -105,8 +106,8 @@ defmodule LoadTest.Communication.Proto.LobbyEventType do
   field(:TYPE_UNSPECIFIED, 0)
   field(:CONNECTED, 1)
   field(:PLAYER_ADDED, 2)
-  field(:GAME_STARTED, 3)
-  field(:START_GAME, 4)
+  field(:START_GAME, 3)
```
Comment (mFragaBA) [inline load_test/lib/load_test/communication/messages.pb.ex:109]: do we still use `START_GAME`?

Comment (agustinesco) [reply load_test/lib/load_test/communication/messages.pb.ex]: You're right! this message was not being used anywhere, deleted

Comment (agustinesco) [reply load_test/lib/load_test/communication/messages.pb.ex]: You're right! this message was not being used anywhere, deleted

**Review by tvillegas98 — DISMISSED** [review]
Nice work @agustinesco ! I think we cannot merge this until [#1214](https://github.com/lambdaclass/curse_of_myrra/pull/1214) is ready to merge 😅.

**Review by ncontinanza — APPROVED** [review]
LGTM!

**Review by mFragaBA — DISMISSED** [review]
LGTM!

**Review by ncontinanza — APPROVED** [review]

**Review by mFragaBA — APPROVED** [review]
LGTM!

**Pull request merged**